frontend/catalyst/utils/tracing.py

```diff
@@ -26,14 +30,26 @@ class TracingContext:
     """
 
     _is_tracing = False
+    _supported_version = "0.4.13"
```

Comment:
@erick-xanadu is there a way we could automate this to avoid forgetting to update it?